src/main/java/org/monarchinitiative/biodownload/DownloadableResource.java

```diff
@@ -0,0 +1,81 @@
+    public static DownloadableResource GO_GAF = new DownloadableResource("goa_human.gaf", createURL("http://geneontology.org/gene-associations/goa_human.gaf"));
+    public static DownloadableResource GO_GAFGZ = new DownloadableResource("goa_human.gaf.gz", createURL("http://geneontology.org/gene-associations/goa_human.gaf.gz"));
+    public static DownloadableResource MONDO_JSON = new DownloadableResource("mondo.json", createURL(".http://purl.obolibrary.org/mondo/mondo.json"));
+    public static DownloadableResource MONDO_OWL = new DownloadableResource("mondo.owl", createURL("http://purl.obolibrary.org/mondo/mondo.owl"));
+    public static DownloadableResource ECTO_JSON = new DownloadableResource("ecto.json", createURL("https://raw.githubusercontent.com/EnvironmentOntology/environmental-exposure-ontology/master/ecto.json"));
+    public static DownloadableResource ECTO_OWL = new DownloadableResource("ecto.owl", createURL("https://raw.githubusercontent.com/EnvironmentOntology/environmental-exposure-ontology/master/ecto.owl"));
+    public static DownloadableResource MAXO_JSON = new DownloadableResource("maxo.json", createURL("https://raw.githubusercontent.com/monarch-initiative/MAxO/master/maxo.json"));
+    public static DownloadableResource MAXO_OWL = new DownloadableResource("maxo.owl", createURL("https://raw.githubusercontent.com/monarch-initiative/MAxO/master/maxo.owl"));
+    public static DownloadableResource MAXO_OBO = new DownloadableResource("maxo.obo", createURL("https://raw.githubusercontent.com/monarch-initiative/MAxO/master/maxo.obo"));
+    public static DownloadableResource HP_JSON = new DownloadableResource("hp.json", createURL("https://raw.githubusercontent.com/obophenotype/human-phenotype-ontology/master/hp.json"));
+    public static DownloadableResource HP_OBO = new DownloadableResource("hp.obo", createURL("https://raw.githubusercontent.com/obophenotype/human-phenotype-ontology/master/hp.obo"));
+    public static DownloadableResource PHENOTYPE_HP = new DownloadableResource("phenotype.hpoa", createURL("http://purl.obolibrary.org/obo/hp/hpoa/phenotype.hpoa"));
+    public static DownloadableResource GENE_INFO = new DownloadableResource("Homo_sapiens_gene_info.gz", createURL("ftp://ftp.ncbi.nih.gov/gene/DATA/GENE_INFO/Mammalia/Homo_sapiens.gene_info.gz"));
+
+    /**
+     * Constructor
+     * @param name Name of file to download to
+     * @param url {@link URL} of source file
+     */
+    public DownloadableResource(String name, URL url) {
+        this.name = name;
+        this.url = url;
+    }
+
+    /**
+     * Create a URL and return null if {@link MalformedURLException} occurs
+     * @param urlStr URL string
+     * @return a {@link URL} or null if malformed
+     */
+    public static URL createURL(String urlStr) {
+        try {
+            return createURLExc(urlStr);
+        } catch(MalformedURLException mue) {
+            logger.error(String.format("Malformed URL for %s", urlStr));
```

Comment:
`String.format` is redundant in logging statements. It is computationally cheaper to use
```
logger.error("Malformed URL for {}", urlStr);
```
because the statement is not generated if the logging level is below the logging threshold. E.g. the user configured the loggers to only show `>=WARN`, and the message is `DEBUG`.
In the case above, the `String.format(...)` is called no matter what.

Reply:
resolved in commit 10b23b9